pylinac/winston_lutz.py

```diff
@@ -115,6 +115,23 @@ def __init__(self, directory: str, use_filenames: bool = False):
         images : :class:`~pylinac.winston_lutz.ImageManager` instance
         """
         self.images = ImageManager(directory, use_filenames)
+        self._json = {}
```

Comment:
Given this call is not expensive, creating a class attr is unnecessary and adds a possible issue of stale data.